include/libpldm/platform.h

```diff
@@ -927,7 +927,7 @@ struct pldm_entity_auxiliary_names_pdr {
 	char auxiliary_name_data[]
 		__attribute__((aligned(alignof(pldm_utf16be))));
 #else
-#error("__has_attribute() support is required to uphold runtime safety")
+#error ("__has_attribute() support is required to uphold runtime safety")
```

Comment:
Commit header seems to be wrong.
please replace this commit with upstream  by cherry-pickhttps://github.com/openbmc/libpldm/commit/a1cd72b9624ce628039716f6244dfc19514bd0fb